<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PictureProcessXClass</class>
 <widget class="QMainWindow" name="PictureProcessXClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PictureProcessX</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="ImageView" name="imageView" native="true"/>
       </item>
       <item>
        <widget class="QWidget" name="widgetToolBox" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>150</height>
          </size>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>50</y>
            <width>231</width>
            <height>24</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QSlider" name="sliderBinarize">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>256</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBinarize">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>256</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget_2">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>80</y>
            <width>231</width>
            <height>24</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QSlider" name="sliderGauss">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinGauss">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QCheckBox" name="checkLight">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>47</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>亮度</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBinarize">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>50</y>
            <width>59</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>二值化</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkGauss">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>80</y>
            <width>71</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>高斯模糊</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkReverseColor">
          <property name="geometry">
           <rect>
            <x>470</x>
            <y>20</y>
            <width>71</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>反相</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkGray">
          <property name="geometry">
           <rect>
            <x>470</x>
            <y>50</y>
            <width>71</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>灰度</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkLivePreview">
          <property name="geometry">
           <rect>
            <x>910</x>
            <y>20</y>
            <width>71</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>实时预览</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkHist">
          <property name="geometry">
           <rect>
            <x>470</x>
            <y>80</y>
            <width>71</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>直方图</string>
          </property>
         </widget>
         <widget class="QWidget" name="">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>20</y>
            <width>231</width>
            <height>24</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QSlider" name="sliderLight">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maximum">
              <number>500</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinLight">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maximum">
              <number>500</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1011</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImageView</class>
   <extends>QWidget</extends>
   <header>ImageView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="PictureProcessX.qrc"/>
 </resources>
 <connections/>
</ui>
